docs-gb/kubernetes/service-meshes/ambassador.md

```diff
@@ -393,7 +393,7 @@ INGRESS_IP
 ```python
```

Comment:
This is python notebook code. The underlying command (without the !) is just bash.

Reply:
fixed